pywr/_core.pyx

```diff
@@ -530,57 +530,21 @@ cdef class Node(AbstractNode):
             self._cost = self._cost_param.value(ts, scenario_index)
 
     cpdef setup(self, model):
```

Comment:
These implementations aren't doing anything other than calling their parent method. Do we still need them?